python/ribasim/ribasim/models.py

```diff
@@ -11,106 +11,117 @@
 
 
 class Edge(BaseModel):
-    remarks: Optional[str] = Field("", description="a hack for pandera")
-    fid: int = Field(..., description="fid")
-    to_node_id: int = Field(..., description="to_node_id")
-    from_node_id: int = Field(..., description="from_node_id")
+    remarks: Optional[str] = Field('', description='a hack for pandera')
```

Comment:
Looks like you still need to run Black after generating this file.
https://deltares.github.io/Ribasim/contribute/addnode.html#finishing-up